packages/taquito-local-forging/src/constants.ts

```diff
@@ -210,6 +210,8 @@ export const opMapping: { [key: string]: string } = {
   '98': 'Lambda_rec',
   '99': 'LAMBDA_REC',
   '9a': 'TICKET',
+  '9b': 'BYTES',
```

Comment:
were these values derived from running the local forging integration tests?

Reply:
no i go them myself

Reply:
https://gitlab.com/tezos/tezos/-/blob/master/docs/mumbai/rpc.rst?plain=1#L3892